src/track/beatgrid.cpp

```diff
@@ -143,41 +143,6 @@ bool BeatGrid::isValid() const {
     return m_sampleRate.isValid() && bpm().isValid() && firstBeatPosition().isValid();
 }
 
-// This could be implemented in the Beats Class itself.
-// If necessary, the child class can redefine it.
```

Comment:
Why write this comment instead of just doing it...

Reply:
@Be-ing I guess you got confused by the diff? This code and comments have been deleted because they became obsolete.

Reply:
I saw that. I am just lamenting that this wasn't already done 10 years ago.